build-system/amp4test.js

```diff
@@ -16,17 +16,21 @@
 'use strict';
 
 const app = module.exports = require('express').Router();
+const minimist = require('minimist');
+const argv = minimist(
+    process.argv.slice(2), {boolean: ['strictBabelTransform']});
+
 
 /* eslint-disable max-len */
 
 /**
- * Logs the given messages to the console in local dev mode, but not while
- * running automated tests.
+ * Logs the given messages to the console when running individual test files
+ * with --files commend line argument.
  * @param {*} messages
  */
 function log(...messages) {
-  if (!process.env.AMP_TEST) {
-    console.log(messages);
+  if (argv.files) {
```

Comment:
I just realized we also have a `verbose` flag. You could use that instead of (or in addition to) `files` if you'd like.

Reply:
SG. switched to `--verbose`